pkg/apis/kubernikus/v1/secret.go

```diff
@@ -103,6 +105,11 @@ type Certificates struct {
 	TLSWormholePrivateKey   string `json:"tls-wormhole-key.pem"`
 	TLSWormholeCertificate  string `json:"tls-wormhole.pem"`
 
+	TLSEtcdCAPrivateKey  string `json:"tls-etcd-ca-key.pem"`
+	TLSEtcdCACertificate string `json:"tls-etcd-ca.pem"`
+	TLSEtcdPrivateKey    string `json:"tls-etcd-key.pem"`
```

Comment:
Shouldn't this be a `EtcdPeersPrivateKey`? Do we even have etcd peers? 🤣

Reply:
This is the server ca for client taking to etcd. its fine. :)